crypto/fipsmodule/hkdf/hkdf.c

```diff
@@ -44,16 +45,26 @@ int HKDF_extract(uint8_t *out_key, size_t *out_len, const EVP_MD *digest,
                  size_t salt_len) {
   // https://tools.ietf.org/html/rfc5869#section-2.2
 
+  // We have to avoid the underlying SHA services updating the indicator
```

Comment:
Small nit, would "HMAC services" be more accurate? Although SHA still is being used under the hood to be fair.